include/fiction/algorithms/simulation/sidb/sidb_simulation_engine.hpp

```diff
@@ -11,7 +11,7 @@ namespace fiction
 /**
  * Selector for the available SiDB simulation engines.
  */
-enum class sidb_simulation_engine
+enum class sidb_simulation_engine : uint8_t
```

Comment:
Make sure `<cstdint>` is included when using fixed-sized integral types.

Reply:
thanks, I have just realized that it is missing.